webapp-django/crashstats/crashstats/forms.py

```diff
@@ -87,7 +87,6 @@ class ReportListForm(BaseForm):
     signature = form_fields.SignatureField(required=True)
     product = forms.MultipleChoiceField(required=False)
     version = forms.MultipleChoiceField(required=False)
-    platform = forms.MultipleChoiceField(required=False)
```

Comment:
Holy moly, I was totally expecting the old TCBS to add it to links to report/list/ when there's an OS selected, but it's not the case, see: https://crash-stats.mozilla.com/topcrasher/products/Firefox/versions/44.0a2/date_range_type/report/crash_type/browser/os_name/Linux/result_count/50?days=7

Well, then I guess that means...